targets/targets.json

```diff
@@ -1276,7 +1276,7 @@
         "inherits": ["Target"],
         "core": "Cortex-M4F",
         "supported_toolchains": ["ARM", "uARM", "GCC_ARM", "IAR"],
-        "extra_labels": ["STM", "STM32F4", "STM32F411RE"],
+        "extra_labels": ["STM", "STM32F4", "STM32F411RE", "DRAGONFLY_MODEM"],
```

Comment:
Do you really need another `TARGET_` directory here? Could you just put it in the `TARGET_MTS_DRAGONFLY_F411RE` directory?